src/igvfd/schemas/changelogs/tabular_file.md

```diff
@@ -2,9 +2,13 @@
 
 ### Minor changes since schema version 11
 
+<<<<<<< HEAD
 * Extend `content_type` enum list to include `differential gene expression quantifications`.
 * Extend `content_type` enum list to include `differential transcript expression quantifications`.
 * Extend `content_type` enum list to include `MPRA sequence designs`.
+=======
+* Add `analysis_step_version`.
+>>>>>>> 9b82ec5f (IGVF-1722-analysis-step-version-schema)
```

Comment:
Looks like this file was messed up during rebase

Reply:
done